docs/platform-services/threat-intelligence-indicators.md

```diff
@@ -418,7 +430,7 @@ If uploading a CSV file with the UI, the format should be the same as used for a
 
 ```
 0001,192.0.2.0,ipv4-addr:value,FreeTAXII,2023-02-21T12:00:00.00Z,2025-05-21T12:00:00.00Z,30,malicious-activity,
```

Comment:
you are missing a comma here, should be
`0001,192.0.2.0,ipv4-addr:value,FreeTAXII,2023-02-21T12:00:00.00Z,2025-05-21T12:00:00.00Z,30,malicious-activity,,`

Reply:
Thanks, @pmontiel-sumo. Fixed.